lib/generation/flutter_project_builder/post_gen_tasks/comp_isolation/isolation_post_gen_task.dart

```diff
@@ -22,4 +22,7 @@ class IsolationPostGenTask implements PostGenTask {
     generationConfiguration.fileStructureStrategy.commandCreated(
         WriteSymbolCommand(null, fileName, isolationCode, symbolPath: 'lib/'));
   }
+
+  @override
+  noSuchMethod(Invocation invocation) => super.noSuchMethod(invocation);
```

Comment:
Is there a reason for this override? It literally just calls the same method from super

Reply:
Ah the reason I let it auto complete it was this or either had it to implement another method from the class that is implementing `PostGenTask` but sure, I can look into a better way

Reply:
Actually with the new changes, I can remove this 👍